<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tables</class>
 <widget class="QWidget" name="Tables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>جدول</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: rgb(228, 229, 244); /* Light gray background */
    border: 1px solid #a9a9a9; /* Darker gray border for a 3D effect */
    border-radius: 6px; /* Rounded corners */
    padding: 5px 10px; /* Padding for a button-like appearance */
    color: black; /* Text color */
    
    /* Subtle gradient for a shiny effect */
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
                                stop:0 #f0f0f0, stop:1 #d3d3d3);
}

QPushButton:hover {
    background-color: #e6e6e6; /* Change to a lighter shade on hover */
    border-color: #b0b0b0; /* Change border color on hover */
}

QPushButton:pressed {
    background-color: #bfbfbf; /* Darker gray when pressed */
    border-color: #808080; /* Slightly darker border when pressed */
}

QPushButton:disabled {
    background-color: #d3d3d3; /* Grayer color for disabled buttons */
    color: #a0a0a0; /* Optional: lighter text color for better contrast */
}
QToolButton {
    background-color: rgb(228, 229, 244); /* Light gray background */
    border: 1px solid #a9a9a9; /* Darker gray border for a 3D effect */
    border-radius: 6px; /* Rounded corners */
     /* Padding for a button-like appearance */
    color: black; /* Text color */
    
    /* Subtle gradient for a shiny effect */
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
                                stop:0 #f0f0f0, stop:1 #d3d3d3);
}

QToolButton:hover {
    background-color: #e6e6e6; /* Change to a lighter shade on hover */
    border-color: #b0b0b0; /* Change border color on hover */
}

QToolButton:pressed {
    background-color: #bfbfbf; /* Darker gray when pressed */
    border-color: #808080; /* Slightly darker border when pressed */
}

QToolButton:disabled {
    background-color: #d3d3d3; /* Grayer color for disabled buttons */
    color: #a0a0a0; /* Optional: lighter text color for better contrast */
}


QWidget {font: 11pt &quot;Segoe UI&quot;;}
QComboBox {
    background-color: #eff3f5; /* Darker grayish background */
    color: #000000; /* Light text color for readability */
    border: 1px solid #5c5c70; /* Slightly lighter border for contrast */
    border-radius: 6px; /* Smooth, rounded edges */
    padding: 5px; /* Padding for a cleaner look */
    font: 11pt &quot;Segoe UI&quot;;
}

QComboBox:focus {
    border-color: #7878a0; /* Lighter border color on focus */
    background-color: #e0e6eb; /* Slightly lighter background on focus */
}
#Tables {
    background-color: #d0dae1; /* Dark background color for the entire widget */
    font: 11pt &quot;Segoe UI&quot;;}
QScrollBar:vertical {
    background: #eff3f5; /* Background color of the scrollbar */
    width: 10px; /* Width of the scrollbar */
    margin: 0px 0px 0px 0px; /* Margins around the scrollbar */
    border-radius: 5px; /* Rounded edges for the scrollbar track */
}

/* Style for the horizontal scrollbar */
QScrollBar:horizontal {
    background: #eff3f5; /* Background color of the scrollbar */
    height: 10px; /* Height of the scrollbar */
    margin: 0px 0px 0px 0px; /* Margins around the scrollbar */
    border-radius: 5px; /* Rounded edges for the scrollbar track */
}

/* Handle (scroller) styling for both horizontal and vertical scrollbars */
QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: #92a8b9; /* Color of the scroller */
    min-height: 20px; /* Minimum height for the vertical handle */
    min-width: 20px; /* Minimum width for the horizontal handle */
    border-radius: 5px; /* Smooth, rounded edges for the handle */
}

QScrollBar::handle:vertical:hover, QScrollBar::handle:horizontal:hover {
    background: #b1c1cd; /* Slightly lighter color on hover for the handle */
}

/* Remove the top/bottom buttons on the scrollbar */
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical,
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none; /* No additional lines/buttons */
    height: 0px;
    width: 0px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical,
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none; /* No page scroll styling */
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="14">
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::RightToLeft</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::EditKeyPressed</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::TextElideMode::ElideRight</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="6" column="0" colspan="14">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: #d0dae1; /* Dark background color for the entire widget */
    font: 11pt &quot;Segoe UI&quot;;
}

/* Style for QLineEdit with a darker background and smoother edges */
QLineEdit {
    background-color: #eff3f5; /* Darker grayish background */
    color: #000000; /* Light text color for readability */
    border: 1px solid #5c5c70; /* Slightly lighter border for contrast */
    border-radius: 6px; /* Smooth, rounded edges */
    padding: 5px; /* Padding for a cleaner look */
    font: 11pt &quot;Segoe UI&quot;;
}

QLineEdit:focus {
    border-color: #7878a0; /* Lighter border color on focus */
    background-color: #e0e6eb; /* Slightly lighter background on focus */
}

QTextBrowser {
    background-color: #eff3f5; /* Darker grayish background */
    color: #000000; /* Light text color for readability */
    border: 1px solid #5c5c70; /* Slightly lighter border for contrast */
    border-radius: 6px; /* Smooth, rounded edges */
    padding: 5px; /* Padding for a cleaner look */
    font: 11pt &quot;Segoe UI&quot;;
}

QTextBrowser:focus {
    border-color: #7878a0; /* Lighter border color on focus */
    background-color: #e0e6eb; /* Slightly lighter background on focus */
}

QCheckBox {
    color: #000000; /* Light text color for dark background */
    font: 11pt &quot;Segoe UI&quot;;
}

QCheckBox:disabled {
    color: #000000; /* Same text color for disabled state */
    opacity: 1; /* Remove opacity reduction */
}

/* Blue indicator for checked and disabled QCheckBox */
QCheckBox::indicator:checked:disabled {
    background-color: #92a8b9; /* Blue color to appear as enabled */
	opacity: 1;
   /* border: 1px solid #3a7bd5;  Blue border to match the indicator */
}

QCheckBox::indicator:disabled {
    opacity: 1; /* Remove opacity change for checkbox indicator */
}

/* Background color for QWidget */
/* Style for the vertical scrollbar */
QScrollBar:vertical {
    background: #eff3f5; /* Background color of the scrollbar */
    width: 10px; /* Width of the scrollbar */
    margin: 0px 0px 0px 0px; /* Margins around the scrollbar */
    border-radius: 5px; /* Rounded edges for the scrollbar track */
}

/* Style for the horizontal scrollbar */
QScrollBar:horizontal {
    background: #eff3f5; /* Background color of the scrollbar */
    height: 10px; /* Height of the scrollbar */
    margin: 0px 0px 0px 0px; /* Margins around the scrollbar */
    border-radius: 5px; /* Rounded edges for the scrollbar track */
}

/* Handle (scroller) styling for both horizontal and vertical scrollbars */
QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: #92a8b9; /* Color of the scroller */
    min-height: 20px; /* Minimum height for the vertical handle */
    min-width: 20px; /* Minimum width for the horizontal handle */
    border-radius: 5px; /* Smooth, rounded edges for the handle */
}

QScrollBar::handle:vertical:hover, QScrollBar::handle:horizontal:hover {
    background: #b1c1cd; /* Slightly lighter color on hover for the handle */
}

/* Remove the top/bottom buttons on the scrollbar */
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical,
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none; /* No additional lines/buttons */
    height: 0px;
    width: 0px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical,
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none; /* No page scroll styling */
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QLineEdit" name="form_Name">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>نام خریدار :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QLineEdit" name="form_SN">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>شماره سریال :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="form_dev">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>نام دستگاه :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLineEdit" name="form_gDate">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>تاریخ انقضای گارانتی :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>184</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLineEdit" name="form_pDate">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>تاریخ خرید :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>125</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::RightToLeft</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>926</width>
           <height>123</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QGroupBox" name="form_parts">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::RightToLeft</enum>
            </property>
            <property name="title">
             <string>قطعات</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="form_partHBox">
               <property name="sizeConstraint">
                <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
               </property>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::RightToLeft</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>926</width>
           <height>98</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="form_bel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::RightToLeft</enum>
            </property>
            <property name="title">
             <string>متعلقات</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="form_belHBox"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>توضیحات</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QTextBrowser" name="form_description">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::RightToLeft</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QPushButton" name="deleteBtn">
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="printBtn">
     <property name="text">
      <string>QR چاپ</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="mainWindowBtn">
     <property name="text">
      <string>صفحه اصلی</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="RefreshBtn">
     <property name="text">
      <string>بازخوانی</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="SearchBtn">
     <property name="text">
      <string>جستجو</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="0" column="12">
    <widget class="QPushButton" name="AddBtn">
     <property name="text">
      <string>افزودن</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="14">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>در حال اضافه کردن دستگاه ها. لطفا منتظر بمانید.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>مشتریان</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>خدمات</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>دستگاه ها</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>محصولات</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QToolButton" name="toolButton">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;خروجی اکسل&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="11">
    <widget class="QPushButton" name="EditBtn">
     <property name="text">
      <string>ویرایش</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>mainWindowBtn</tabstop>
  <tabstop>RefreshBtn</tabstop>
  <tabstop>SearchBtn</tabstop>
  <tabstop>deleteBtn</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
